tests/javascript_tests/test_shape_controller.js

```diff
@@ -36,7 +36,11 @@ requirejs([
         decomp = new DecompositionModel(name, ids, coords, pct_var, md_headers,
             metadata);
         var dv = new DecompositionView(decomp);
-        this.sharedDecompositionViewDict.pcoa = dv;
+        // Add shapes to the initial markers
+        for (i = 0; i < dv.markers.lengthl; i++) {
```

Comment:
nope, not after the change made to load JSON better. Entire chunk removed again.